layouts/shortcodes/install-python.html

```diff
@@ -12,4 +12,3 @@
         various operating systems and distributions.
     </p>
 </div>
-
```

Comment:
We probably want to keep this. (VSCode should have a setting for automatically inserting end-of-file newlines.)